registry/polkadot.json

```
@@ -381,5 +381,13 @@
         "shortDescription": "Web3 Delegation",
         "longDescription": "The Sons of Gavin is operated by TheeWeb3Patriot, It is a web3 centered delagate that holds values of being treasury conservatives.",
         "isOrganization": false
+    },
+  {
+        "address": "1jPw3Qo72Ahn7Ynfg8kmYNLEPvHWHhPfPNgpJfp5bkLZdrF",
+        "name": "Jimmy Tudeski - Stakenode",
+        "image": "https://github.com/jimmy-tudeski/opengov-delegate-registry/blob/master/images/Jimmy%20Profile%20.png",
```

use link to our repo, please
```suggestion
        "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/Jimmy%20Profile%20.png",
```